extensions/omni_httpc/omni_httpc.c

```diff
@@ -222,18 +230,22 @@ struct request {
   int version;
   // Signals that the authority has been connected to for this request
   bool connected;
+  // Signals that the request has finished processing
+  bool complete;
```

Comment:
this kind of duplicates the current `done` counter, but it's needed to check if the redirect request is done yet.